md/bpm-api.md

```diff
@@ -4543,3 +4543,184 @@ Specify the next execution date of a timer event trigger.
     ```
     1433980484194
     ```
+
+### ProcessSupervisor
+
+#### Description
+
+The process supervisor has management rights over a process. He can configure and monitor it.
+You can give the ProcessSupervisor rights to some users by specifying a role and or a group, or a specific user.
+In order to be able to manage the processes he supervises in the portal, a user should also have the profile "Process Manager".
+
+#### Identifier
+
+A compound identifier constructed from process\_id/user\_id/role\_id/group\_id where all Ids are long values.
+
+#### Representation
+```json
+{
+  "process_id":"Id of the process",
+  "role_id":"Id of role, or -1 if the supervisor type is not role or membership",
+  "group_id":"Id of group, or -1 if the supervisor type is not group or membership",
+  "user_id":"Id of user, or -1 if the supervisor type is not user"
+}
+
+```
+#### Methods
+
+* POST - Add a process supervisor
+* GET - Search for process supervisors
+* DELETE - Remove a process supervisor
+
+<a id="process-supervisor-deploy"/>
+
+#### Response object extension (deploy query parameter)
+
+The `d` (deploy) used to [extend response object](rest-api-overview.md#extend-resource) can be used with : 
+  * `role_id`
+  * `group_id`
+  * `user_id`
+
+#### Search for process supervisors of a given type (user, group, role or membership)
+
+* **URL**  
+  `/API/bpm/processSupervisor`  
+_Example_: Get the supervisors of type `User` for the process 8040901857674754544: `API/bpm/processSupervisor?c=5&d=user_id&f=process_id%3D8040901857674754544&f=user_id%3D>0&f=group_id%3D-1&f=role_id%3D-1&p=0`
+* **Method**  
+  `GET`
+* **Data Params**  
+  [Standard search parameters](rest-api-overview.md#resource_search) are available.  
+  * f: filter of the search, you need to specify the process_id, and then the user\_id, group\_id and role\_id with one of them (two if you want to filter on group and role) set to `>0` and the other ones set to `-1`. E.g.: `f=process_id%3D8040901857674754544&f=user_id%3D>0&f=group_id%3D-1&f=role_id%3D-1`
+  * d: extend resource response parameters of [this resource](#process-supervisor-deploy) are available.
+* **Success Response**  
+  JSON representations of matching process supervisors
+  * **Code**: 200
+  * **Payload**:  
+    ```json
```

Comment:
Little typo here... When i try to see this in docker on PR. I don't see the content of this part.